js/heatmap.js

```diff
@@ -119,6 +115,55 @@ window.addEventListener('DOMContentLoaded', () => {
       keys.forEach((keyName) => { keyFinger[keyName] = f; });
     });
 
+    const requiresExtension = keyCode => (
+      Array.from("TGBNHY").some(l => l == keyCode.at(3)) ||  // IndexInner
+      !(  // Not in 3×10 matrix
+        keyCode.startsWith('Key')
+        || ['Space', 'Comma', 'Period', 'Slash', 'Semicolon'].includes(keyCode)
+      )
+    );
+
+    const getKeyRow = keyCode => {
+      if (keyCode == "Space") return 0;
```

Comment:
any reason why you’re using `==` instead of `===` ?

(nitpick, non-consistent quote style, you should probably stick to single quotes)

Reply:
Oh I just forgot that `[] == ""` = `true` was a thing

(As for the quotes, I have another PR that ran prettier on the code base, but I forgot to actually merge it)